server/src/main/resources/application-qa.properties

```
@@ -10,7 +10,6 @@ kitu.kotoutumiskoulutus.koealusta.scheduling.enabled=true
 kitu.kotoutumiskoulutus.koealusta.scheduling.import.schedule=FIXED_DELAY|3600s
 
 kitu.oppijanumero.username=koto-rekisteri
```

Pitäiskö tässä olla joku arvo, esim:
```suggestion
kitu.oppijanumero.username=koto-rekisteri
kitu.oppijanumero.callerid=1.2.246.562.24.77101904300.koto-rekisteri
```

---

ja sama varmaan tarvitaan `server/src/main/resources/application-dev.properties` - filulle

---

Nää tulee oletuksena `application.properties`-filusta jos ei oo erikseen määritelty ympäristökohtaisia. Laitoin sinne, kun näissä käytetään kaikissa samaa. Periaatteessa ton usernamenkin voisi siirtää kokonaan sinne.

---

Joo 👍